nixos/tests/qtile/default.nix

```diff
@@ -0,0 +1,70 @@
+import ../make-test-python.nix ({ lib, ...} : {
+  name = "qtile";
+
+  meta = {
+    maintainers = with lib.maintainers; [ sigmanificient ];
+  };
+
+  nodes.machine = { pkgs, lib, ... }: let
+    # We create a custom Qtile configuration file that adds a widget from
+    # qtile-extras to the bar. This ensure that the qtile-extras package
+    # also works, and that extraPackages behave as expected.
+
+    config = { stdenvNoCC, fetchurl }:
+      stdenvNoCC.mkDerivation {
+        name = "qtile-config";
+        version = "0.0.1";
+
+        src = fetchurl {
+          url = "https://raw.githubusercontent.com/qtile/qtile/master/libqtile/resources/default_config.py";
```

Comment:
```suggestion
          url = "https://raw.githubusercontent.com/qtile/qtile/SOME_REVISION/libqtile/resources/default_config.py";
```

Reply:
should I use `${qtile.srv.rev}` or just hard code something like `v0.28.1`?